src/Stripe.net/Entities/Events/Event.cs

```diff
@@ -4,36 +4,73 @@ namespace Stripe
     using Newtonsoft.Json;
     using Stripe.Infrastructure;
 
+    [JsonConverter(typeof(EventConverter))]
     public class Event : StripeEntity, IHasId, IHasObject
     {
+        /// <summary>
+        /// Unique identifier for the object.
+        /// </summary>
         [JsonProperty("id")]
         public string Id { get; set; }
 
+        /// <summary>
+        /// String representing the object’s type. Objects of the same type share the same value.
+        /// </summary>
         [JsonProperty("object")]
         public string Object { get; set; }
 
+        /// <summary>
+        /// For Connect events, the identifier of the account that emitted the event.
+        /// </summary>
         [JsonProperty("account")]
         public string Account { get; set; }
 
+        /// <summary>
+        /// The Stripe API version used to render <c>data</c>.
+        /// </summary>
+        /// <remarks>
+        /// Note: This property is populated only for events on or after October 31, 2014.
+        /// </remarks>
         [JsonProperty("api_version")]
         public string ApiVersion { get; set; }
```

Comment:
realizing this might affect your version mismatch error handling no?

Reply:
Hm, good point. I don't think this is going to be a huge problem in practice since `EventUtility.ParseEvent` should only be used when receiving a webhook though.